src/Core/Content/Category/Aggregate/CategoryTranslation/CategoryTranslationEntity.php

```diff
@@ -17,37 +17,37 @@ class CategoryTranslationEntity extends TranslationEntity
 
     protected string $categoryVersionId;
 
-    protected ?string $name;
+    protected ?string $name = null;
 
     /**
      * @var array<string>|null
      */
-    protected ?array $breadcrumb;
+    protected ?array $breadcrumb = [];
```

Comment:
that is changing the previous behaviour. before it was default `null`, now an array. that can lead to unexpected behaviour

Reply:
ok, i will change the default to null also for arrays